app/views/layouts/_star.html.erb

```diff
@@ -0,0 +1,36 @@
+<span class="stars">
+  <% case model
```

Comment:
I love the way you used a partial for this. You could utilize a loop that would generate the stars for the number specified and that would really simplify this view.